system-test/read-rows.ts

```diff
@@ -85,7 +85,7 @@ describe('Bigtable/Table', () => {
   const TABLE = INSTANCE.table('table');
 
   describe('close', () => {
-    it('should fail when invoking readRows with closed client', async () => {
+    it.skip('should fail when invoking readRows with closed client', async () => {
```

Comment:
can we add retries to this test instead? I would prefer not to skip it, unless it is consistently failing.

Reply:
Yes. I added a change which will still allow the test to run and retry in the appropriate circumstances.